src/Appwrite/Platform/Tasks/DeleteOrphanedProjects.php

```diff
@@ -110,6 +115,15 @@ public function action(Group $pools, Cache $cache, Database $dbForConsole, Regis
             $count = $count + $sum;
         }
 
-        Console::log('Iterated through ' . $count - 1 . '/' . $totalProjects . ' projects found ' . $orphans . ' orphans');
+        Console::log('Iterated through ' . $count - 1 . '/' . $totalProjects . ' projects found ' . $orphans - 1 . ' orphans');
+    }
+
+    private function deleteProject(Database $dbForConsole, $projectId): void
+    {
+        try {
+            $dbForConsole->deleteDocument('projects', $projectId);
+        } catch (\Throwable $th) {
+            Console::error('Error when trying to delete project (' . $projectId . ') ' . $th->getMessage());
+        }
```

Comment:
You probably also need to delete the other tables in the project's namespace.

Reply:
Not sure I understand what you mean by "other tables in the project's namespace"
Do you mean the metadata collection if it was created?

Reply:
No, its possible that certain tables of the project are still created like _buckets _databases _users. so we've to check if those tables exist and delete them too